page_object/toDoListPageObject.ts

```diff
@@ -0,0 +1,48 @@
+import {expect} from '@playwright/test';
+
+const ulrTodolist = 'https://webdriveruniversity.com/To-Do-List/index.html'
+const newTodoPlaceholder = 'Add new todo';
+
+export class ToDoListPageObject{
+    async navigate(page: any){
+        await page.goto(ulrTodolist)
+    }
+
+    async checkPageTitle (page: any){
```

Comment:
We use test.step in our project. Please add to all method test.step to check how it works.